test/common/include/alpaka/test/dim/TestDims.hpp

```diff
@@ -34,9 +24,7 @@ namespace alpaka
             using TestDims =
                 std::tuple<
                     alpaka::dim::DimInt<1u>
-#if !defined(ALPAKA_CUDA_CI)
                     ,alpaka::dim::DimInt<2u>
-#endif
```

Comment:
We did not test the 2D CUDA case before.
The reason was that the builds timed out years ago. This seems to be no issue anymore.